### ISSUE TITLE
High-level APIs for testing a video and rawframes not working

### ISSUE THREAD

**Issue by sagilizbona** [open-mmlab/mmaction2#1568]
Hi, 

I have tried to run the first example in Getting Started High-level APIs for testing a video and rawframes:

> "Here is an example of building the model and testing a given video"

and got the following issue:

```

Traceback (most recent call last):
  File "/home/mmaction2/mmaction/core/hooks/output.py", line 44, in register
    layer = rgetattr(module, name)
  File "/home/mmaction2/mmaction/core/hooks/output.py", line 68, in rgetattr
    return functools.reduce(_getattr, [obj] + attr.split('.'))
  File "/home/mmaction2/mmaction/core/hooks/output.py", line 66, in _getattr
    return getattr(obj, attr, *args)
  File "/home/anaconda3/envs/open-mmlab/lib/python3.7/site-packages/torch/nn/modules/module.py", line 1186, in __getattr__
    type(self).__name__, name))
AttributeError: 'Recognizer2D' object has no attribute 'tools/data/kinetics/label_map_k400'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/mmaction2/main.py", line 34, in <module>
    results = inference_recognizer(model, video, labels)
  File "/home/mmaction2/mmaction/apis/inference.py", line 179, in inference_recognizer
    with OutputHook(model, outputs=outputs, as_tensor=as_tensor) as h:
  File "/home/mmaction2/mmaction/core/hooks/output.py", line 23, in __init__
    self.register(module)
  File "/home/mmaction2/mmaction/core/hooks/output.py", line 47, in register
    raise AttributeError(f'Module {name} not found')
AttributeError: Module tools/data/kinetics/label_map_k400.txt not found

```

I have seen that line:
`results = inference_recognizer(model, video, labels)`
is from an old version but i wonder if there is a new test example?

**Comment by Dai-Wenxun** [comment]
Hi, @sagilizbona, 
just remove the `labels` in `inference_recognizer()` and it works very well. The `inference_recognizer()` function does not need the `labels`. You can pull request to update it.

**Comment by kennymckormick** [comment]
Solved in #1580
